packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -94,13 +94,13 @@ module.exports = (resolve, rootDir, isEjecting) => {
     const unsupportedKeys = Object.keys(overrides);
     if (unsupportedKeys.length) {
       const isOverridingSetupFile =
-        unsupportedKeys.indexOf('setupTestFrameworkScriptFile') > -1;
+        unsupportedKeys.indexOf('setupFilesAfterEnv') > -1;
```

Comment:
hey @loryman 

there is list of supported keys

https://github.com/facebook/create-react-app/blob/537c115217cccd70e451a8268c3762df82d0c4b6/packages/react-scripts/scripts/utils/createJestConfig.js#L76-L86

Can I ask you to add `extraGlobals` key there?

It drastically improves performance of jest image snapshot test

jest docs: https://jestjs.io/docs/en/configuration#extraglobals-array-string

Isssues: https://github.com/facebook/jest/issues/5163

Reply:
@iansu Any thoughts on this?

Reply:
I think it's probably okay to add that to the supported keys.

Reply:
Thank you!